sql/catalyst/src/test/scala/org/apache/spark/sql/types/StructTypeSuite.scala

```diff
@@ -73,6 +73,29 @@ class StructTypeSuite extends SparkFunSuite with SQLHelper {
     assert(struct.toDDL == """`b` BOOLEAN COMMENT 'Field\'s comment'""")
   }
 
+  private val nestedStruct = new StructType()
```

Comment:
I would add a test with empty space field since we found that it might matter :-)